test/spec_helper.rb

```diff
@@ -22,12 +22,26 @@
 
 require "yast"
 
-# stub module to prevent its Import
-# Useful for modules from different yast packages, to avoid build dependencies
+# Stubs module to prevent its Import
+#
+# Useful for modules from different yast packages, to avoid build dependencies.
+#
+# @param name [#to_sym] module name
 def stub_module(name)
+  drop_module(name) if Yast.const_defined?(name.to_sym)
```

Comment:
ugh, what why? if module is defined, then why you need to stub it?

Reply:
Umm, because if the module is defined and I tried to stub it in some test (because whatever reason), the stub method does nothing and this could make confusing. I would expect a stub object when I use `stub_module` method.

Reply:
well, in other module it do nothing if module is already loaded and only "drawback" is stronger test when module is available.